constants/index.ts

```diff
@@ -262,4 +262,6 @@ export const MAX_DAILY_EMAILS = 100 // If changed, update the DB default accordi
 export const XEC_TX_EXPLORER_URL = 'https://explorer.e.cash/tx/'
 export const BCH_TX_EXPLORER_URL = 'https://blockchair.com/bitcoin-cash/transaction/'
 
-export const MAX_MESSAGES_TO_PROCESS_AT_A_TIME = 2
+export const MAX_MEMPOOL_TXS_TO_PROCESS_AT_A_TIME = 3
+export const CHRONIK_INITIALIZATION_DELAY = 500
+export const MEMPOOL_PROCESS_DELAY = 100
```

Comment:
Why not 0?

Reply:
If there is more than `MAX_MEMPOOL_TXS_TO_PROCESS_AT_A_TIME`, the thread waits in a loop and checks it every `MEMPOOL_PROCESS_DELAY` miliseconds to try again. Using at 0 would add unecessary overhead, since it would check it many times without any changes. I changed the constant name to `MEMPOOL_BLOCKAGE_LOOP_DELAY` to improve the description of what it does.